src/Microsoft.VisualStudio.ProjectSystem.VisualBasic.VS/ProjectSystem/VS/LanguageServices/VisualBasicVsContainedLanguageComponentsFactory.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.ComponentModel.Composition;
+using Microsoft.VisualStudio.Shell;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.LanguageServices
+{
+    [Export(typeof(IVsContainedLanguageComponentsFactory))]
+    [AppliesTo(ProjectCapability.VisualBasic)]
+    internal class VisualBasicVsContainedLanguageComponentsFactory : VsContainedLanguageComponentsFactoryBase
```

Comment:
In the future, we're differentiating between C# and VB in the targets files - instead of having a separate component for each. I'll move this one to that model when I move the others.